### ISSUE TITLE
CSV File not being imported

### ISSUE THREAD

**Issue by FadyEffat** [magespecialist/m2-MSP_CashOnDelivery#13]
I try to put a file for the rules but it's not being uploaded M2.1.2

**Comment by FadyEffat** [comment]
Update to this issue , I manually inserted the rules in the mysql db now rules are visible but it causes the checkout to freeze

**Comment by phoenix128** [comment]
Hello @FadyEffat ,
do you have any error on your browser console or php error log?

**Comment by FadyEffat** [comment]
It's empty

**Comment by phoenix128** [comment]
Unless you have other problems with your checkout you must have some error, at least in your PHP log. Does you checkout wotk if you disable the module?

**Comment by Tristan-N** [comment]
I have the same issue. After some checking I noticed the following. First, the screenshot is the readme from this module. I noticed that in my database the columns: msp_cod_id AUTO increment (or something like that), Country, From_Amount, Free and is_pct can be found. This means there is no website column. However, while uploading the CSV file the error that occurs tells me to add the website column. When I add it in the CSV, I get the error that the column website can't be found. 

Somehow there is a difference between the PHP and the database. Mostly for the column 'website'. 
If I have more information I'll let you know. 

![schermafbeelding_2017-01-04_om_09_33_22](https://cloud.githubusercontent.com/assets/19165453/21636016/270e62e2-d261-11e6-8be1-48fbae4ec612.png)
